core/src/main/java/com/adyen/checkout/core/common/internal/ui/CheckoutNetworkLogo.kt

```diff
@@ -58,7 +64,17 @@ fun CheckoutNetworkLogo(
 
     NetworkImage(
         url = fullUrl,
-        modifier = modifier,
+        modifier = modifier
```

Comment:
Figma suggests to use this 👇 Didn't test it, but it seems like it would create this harder shadow to define the border first and add a wider softer shadow after.
```Kotlin
Modifier
    .shadow(elevation = 2.dp, spotColor = Color(0x05001222), ambientColor = Color(0x05001222))
    .shadow(elevation = 4.dp, spotColor = Color(0x0A001222), ambientColor = Color(0x0A001222))
    .padding(0.dp)
    .width(24.dp)
    .height(16.dp)
```